portfolio/src/main/java/com/google/sps/authentication/servlets/LoginServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.authentication.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
```

Comment:
Let's make it final class. And also add some public javadoc.

Reply:
Fixed